com.unity.render-pipelines.core/Editor/Camera/CameraUI.Output.Drawers.cs

```diff
@@ -0,0 +1,26 @@
+namespace UnityEditor.Rendering
+{
+    /// <summary> Camera UI Shared Properties among SRP</summary>
+    public static partial class CameraUI
+    {
+        public partial class Output
```

Comment:
Missing API docs for this one?

Reply:
Thank you, I passed the doc checker to the core package and I miss this :(.